linode/manager
# M3-5664: Prevent previous saved support text from loading into a reply

## Description

This PR prevents the saved text from the initial support ticket creation from autofilling the reply textfield.

The next step is to only autofill the reply textfield if the text is related to that reply which will be addressed in a future ticket.

## How to test

1. Open a new ticket
2. Respond to that ticket
3. Reload the page and the textfield should no longer be autofilled
